SCClassLibrary/Common/Core/Object.sc

```diff
@@ -389,6 +389,7 @@ Object  {
 	dereference { ^this } // see Ref::dereference
 	reference { ^Ref.new(this) }
 	asRef { ^Ref.new(this) }
+	dereferenceOperand { ^this }
```

Comment:
I don't see the need for this definition. We can avoid blind calls through `isKindOf`/`respondsTo`.